tests/examples/dynamic-provider/secrets/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "minimal",
+    "license": "Apache-2.0",
+    "devDependencies": {
+        "typescript": "^3.0.0"
+    },
+    "peerDependencies": {
+        "@pulumi/pulumi": "latest"
+    },
+    "dependencies": {
+        "@pulumi/pulumi": "^3.73.0"
+    }
```

Comment:
Nit: I'm guessing this was leftover from local testing? Can we remove `"dependencies"` since we have it listed in `"peerDependencies"`?